lib/aur-chroot

```diff
@@ -124,6 +147,9 @@ while true; do
     shift
 done
 
+printf >&2 $'You\'re using a new version of aur-chroot.\n'
```

Comment:
In diagnostic messages try to avoid contractions. Consider something like `Set AUR_BUILD_CHROOT=chroot if experiencing problems with this newer implementation of aur-chroot.`

However it seems like this kind of message which will eventually be removed in time so I wonder if it's needed at all in the code instead of being part of the release notes.

Reply:
> However it seems like this kind of message which will eventually be removed in time so I wonder if it's needed at all in the code instead of being part of the release notes.

I've changed the code according to the first part of this comment. On the second part though I feel like this is a pretty big change and nobody really reads changelog. One line sounds like a fair price to avoid N issues with people complaining their build broke.

Reply:
Usually with these kind of major changes I put a message in the `.install` file of the (AUR) PKGBUILD and write some announcement on the forums and AUR comments. That being said, I'd still keep the old `aur-chroot` as default for the time being. Then if it's been sufficiently been tested by people using chroot, it can be made the default.